components/ipfs/ipfs_utils.cc

```diff
@@ -583,4 +586,12 @@ absl::optional<GURL> ExtractSourceFromGateway(const GURL& url) {
   return ExtractSourceFromGatewayPath(url);
 }
 
+static jboolean JNI_BravePrivacySettingsIPFSUtils_IsIPFSSupported(JNIEnv* env) {
```

Comment:
How about to expand 
https://github.com/brave/brave-core/blob/master/build/android/java/templates/BraveConfig.template
and get rid of this method and whole `android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettingsIPFSUtils.java` file ?

Reply:
actually, don't remove `BravePrivacySettingsIPFSUtils.java` as it may be useful for other https://github.com/brave/brave-core/pull/16752/files#r1081858479

Reply:
we really better use https://github.com/brave/brave-core/blob/master/build/android/java/templates/BraveConfig.template as Alexey mentioned for that check at least with a preprocessor definition. No need to go back and force to native for that, it will be set in `BraveConfig.java` during a build and that's it

Reply:
Yeah I didn't realize there is a template way to write Java code, updating now.

Reply:
Updated now.